src/main/java/Console.java

```diff
@@ -144,32 +144,31 @@ public static String NewUserMenu(Scanner reader, User user) throws Exception {
         UserAnalyzer analyzer = COMMANDS.get(command);
         RunCommand commandExecutor = new RunCommand(analyzer);
         if (command == 4){
+            commandExecutor.resetPotentialDisease();
+            commandExecutor.changeInfo(user, new ArrayList<String>(), 3);
             int potentialDisease = 1000;
             ArrayList<String> currentSymptoms = new ArrayList<>();
             while(potentialDisease > 6) {
-//                    System.out.println("While while while");
                     potentialDisease = commandExecutor.executeCommandDisease(user, currentSymptoms);
                     if(potentialDisease <= 6){
-//                        System.out.println("Breaking");
-//                        System.out.println("blahe:" + potentialDisease);
-
                         break;
                     }
                     //outputs how many potential disease client could have
                     // pass in empty array list if it is first round.
-                    System.out.println("These are your options:");
+                    System.out.println("These are your options. If none of them apply to you, please type in N/A.");
                     Presenter analyze_results = new Presenter(analyzer);
                     System.out.println(analyze_results.retrieveOutput()); //first time giving options
                     String symptoms = reader.nextLine(); //client's input of symptoms
+                    if(symptoms.equals("N/A")){}
+                    else{
```

Comment:
Why don't we just change the whole if-else block into "if (!symtoms.equals("N/A"){}"?